### DISCUSSION TITLE
performance issue (vuetify 2) probably when rendering

### DISCUSSION THREAD

**Discussion by guykatz** [vuetifyjs/vuetify#13851] [Q&A]
hi;
this is a somewhat generic question.
In most of my app screens I have a title/subtitle. and dividing them its a gradient css that is a vuetify linear loading bar (it animates while loading and stops animating after data is loaded.
 For med to large amount or data that is loaded form the server, the UX I encounter is that the loading animation suddenly gets stuck for some time - I am not sure at which phase but maybe in rendering.
I was wondering what I can do in order to improve on this UX. any ideas?
thanks,
BTW, the data is not that large is a up to a couple of hundreds of records in a data table (that I cannot do paging with - due to the nature of the application)

![image](https://user-images.githubusercontent.com/2852437/123600331-3de54500-d7ff-11eb-8afd-9911b216fa16.png)

**Comment by KaelWD** [comment]
If you only animate `transform` and `opacity` it will run on a separate thread and won't be blocked by data processing or rendering.

**Reply by guykatz** [reply]
thanks @KaelWD I will try and take a look and hope for the best :)
